packages/admin/admin/src/table/paging/createOffsetLimitPagingAction.ts

```diff
@@ -22,9 +22,10 @@ export function createOffsetLimitPagingAction<TData extends OffsetLimitPagingDat
     };
     const createFetchPreviousPage = () => {
         const previousPage = currentPage - 1;
+        const newOffset = pagingApi.current - limit >= 0 ? pagingApi.current - limit : 0;
```

Comment:
In the DataGrid it's possible to change the pageSize. This lead to the offset having negative values because the limit changed (e.g. from 10 to 20).

However, I did not investigate close enough. The root cause of this is that the offset is not updated correctly. I'll fix that now.